docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+
+Prometheus can scrape metric data from software and infrastructure and store it. Scraping means that Prometheus software periodically revisits the same endpoint to check for new data. Prometheus “scrapes” data from a piece of software that is instrumented with a client library.
+
+For example, a NodeJS application can configure the [prom-client](https://github.com/siimon/prom-client) to expose metrics easily at an endpoint, and Prometheus can regularly scrape that endpoint. There are a number of other tools available within the Prometheus toolkit that you can use to instrument applications.
+
+## Prometheus as deployment
+
+The first section of this document introduced the _Prometheus as Data_ concept and how the Prometheus data model and metrics are organized. The second section introduced the concept of _Prometheus as Software_ that is used to collect, process, and store metrics.
+
+This section describes how “Prometheus as Data" and “Prometheus as Software” come together. Suppose an application called `MyApp` uses a Prometheus client to expose metrics. One approach to collecting metrics data is to use a URL in the application that points to an endpoint `http://localhost:3000/metrics` that produces Prometheus metrics data.
+
+The following image shows the two metrics associated with the endpoint. The HELP text explains what the metric means, and the TYPE text indicates what kind of metric it is (in this case, a gauge). `MyAppnodejs_active_request_total` indicates the number of requests (in this case, `1`). `MyAppnodejs_heap_size_total_bytes` indicates the heap size reported in bytes. There are only two numbers because this data shows the value at the moment the data was fetched.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/endpoint-data.png" max-width="750px" caption="Endpoint example" >}}
+
+These metrics are available in an HTTP endpoint, but how do they get to Grafana, and subsequently, into a dashboard?
+
+That’s where you can use either the Prometheus software, or the [Grafana Agent](/docs/agent/latest/). The Grafana Agent collects and forwards telemetry data to open sourcevdeployments of the Grafana Stack, Grafana Cloud, or Grafana Enterprise, where your data can then be analyzed.
+
+Telemetry refers to the process of recording and transmitting the readings of an application or piece of infrastructure. Telemetry is critical to observability because it helps you understand exactly what's going on in your infrastructure. Telemetry data is a source of truth.
+
+Those metrics above, for example, `MyAppnodejs_active_requests_total`, is telemetry data. MyApp only makes it available for pull by means of an HTTP request. You can configure the Grafana Agent to pull that data from MyApp every 5 seconds, and send the results to Grafana Cloud.
```

Comment:
Sheesh, this is just complicated, I'm the author and I don't know the best way to explain this.  So let me break it down in the not simple way and then beg for help to explain it in the simple way.

OK so as discussed in the article, Prometheus can be used to collect & forward metrics to Grafana.  So _technically_ you don't need Grafana Agent at all!  But here's the thing:  grafana agent is engineered to be easier to use.  And *we* know stuff about their later learning journey (namely that later on we'll talk to them about logs and traces) that Prometheus doesn't easily do.   So we're setting them up for success and a smoother learning journey by introducing Grafana Agent now as the best way to do these things.  While they _could_ use Prometheus, if we didn't swtich topics, we'd make this text easier to follow now at the expense of a harder learning journey later.

We want to introduce grafana agent *now* because it uses Prometheus, and because it's a core bit of instrumentation for all types of observability, many of which _we aren't talking about in this article_.   That's the truth.  How best to write that in?  I need help.